specfile/specfile.py

```diff
@@ -530,7 +530,7 @@ def release(self, value: str) -> None:
     @property
     def expanded_release(self) -> str:
```

Comment:
Shouldn't it be called `expanded_release_without_dist()` because I'd expect it returns `28.fc37` rather than `28` (unless I read the docstring first)

Reply:
I decided to go with `release`/`expanded_release` and `raw_release`/`expanded_raw_release` pairs because the former will be used way more often and it's convenient to simply write for example `spec.release = 28`, without worrying about `%dist`.

Reply:
On the other hand it's true that `spec.release_without_dist` would be completely self-explanatory.

Reply:
I've opened https://github.com/packit/specfile/issues/148.